### PR TITLE
Add acceptance test scenario to verify `nodejs_postgresql_nameaces` example

### PR DESCRIPTION
**Pull request by pratikjagrut**
### Motivation
JIRA TICKET: https://issues.redhat.com/browse/APPSVC-706

### CODE REVIEW

File: test/acceptance/features/examples.feature [hunk 1]
```diff
@@ -7,6 +7,7 @@ Feature: Verify examples provided in Service Binding Operator github repository
     Background:
         Given Namespace [TEST_NAMESPACE] is used
         * Service Binding Operator is running
+        * PostgreSQL DB operator is installed
```
Comment (pedjak) [inline test/acceptance/features/examples.feature:10]: this would mean that each scenario need postgresql operator, but it does not. Hence, this step should be moved into the scenario.

File: test/acceptance/features/examples.feature [hunk 2]
```diff
@@ -53,3 +54,37 @@ Feature: Verify examples provided in Service Binding Operator github repository
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "sbr-to-bind-hello-app-to-route" should be changed to "True"
         And jq ".status.conditions[] | select(.type=="Ready").status" of Service Binding "sbr-to-bind-hello-app-to-route" should be changed to "True"
         And Secret "sbr-to-bind-hello-app-to-route" contains "ROUTE_HOST" key with value "example-sbo.apps.ci-ln-smyggvb-d5d6b.origin-ci-int-aws.dev.rhcloud.com"
+
+    # https://github.com/redhat-developer/service-binding-operator/tree/master/examples/nodejs_postgresql_namespaces
+    Scenario: Binding an Imported app in one namespace to an In-cluster Operator Managed PostgreSQL Database in another namespace
```
Comment (pedjak) [inline test/acceptance/features/examples.feature:59]: how about to rephrase it to:

```
Bind application to a database running in another namespace
```

**Review by pedjak — CHANGES_REQUESTED** [review]

File: test/acceptance/features/examples.feature (hunk 1 above)
Comment (pratikjagrut) [reply test/acceptance/features/examples.feature:10]: There are upcoming examples which need postgresql operator. One of them is PR #768.

File: test/acceptance/features/examples.feature (hunk 2 above)
Comment (pratikjagrut) [reply test/acceptance/features/examples.feature:59]: The scenario name deliberately kept the same as example name. The suggested name sounds cool, should I also update the [example name](https://github.com/redhat-developer/service-binding-operator/blob/master/examples/nodejs_postgresql_namespaces/README.md#binding-an-imported-app-in-one-namespace-to-an-in-cluster-operator-managed-postgresql-database-in-another-namespace)?

Comment (pedjak) [reply test/acceptance/features/examples.feature:59]: Yes, please.

Comment (pratikjagrut) [reply test/acceptance/features/examples.feature]: Done. PTAL.

**Review by pedjak — APPROVED** [review]
please squash

**Comment by pratikjagrut** [comment]
Squashed.
